example/README.md

```diff
@@ -0,0 +1,228 @@
+Getting Started
+===============
+
+## Quick Start with kind
+
+You can see a demonstration of how TopoLVM provisioner works with the following commands. This demonstration using [kind](https://github.com/kubernetes-sigs/kind) and loopback device on your host.
+```console
+$ cd example
+$ make setup run
+```
+
+You can see Logical Volumes are attached to PV as follows:
```

Comment:
TopoLVM provision an LVM logical volume and bind it with a PersistentVolumeClaim as follows: